### PR TITLE
daemon: return immediately if job run without verbose

### PR DESCRIPTION
**Pull request by FeelyChau**
return immediately if job run without verbose.

### CODE REVIEW

File: packages/daemon/src/app/controller/job.ts [hunk 1]
```diff
@@ -60,15 +61,18 @@ export class JobController {
         sse.finish();
       }
     } else {
-      try {
-        const plugins = await this.pipelineService.installPlugins(job, cwd, pyIndex);
+      const run = async () => {
```
Comment (yorkie) [inline packages/daemon/src/app/controller/job.ts:64]: Why do you create this function and it gets called once.

Comment (FeelyChau) [reply packages/daemon/src/app/controller/job.ts]: Moved into `nextTick`

File: packages/daemon/src/app/controller/job.ts [hunk 2]
```diff
@@ -60,19 +61,21 @@ export class JobController {
...
-        return failRes(this.ctx, {
-          message: err?.message,
-          data: job
-        });
-      }
       successRes(this.ctx, {
         message: 'create pipeline and jobs successfully',
         data: job
       }, 201);
+      process.nextTick(async () => {
```
Comment (yorkie) [inline packages/daemon/src/app/controller/job.ts:68]: It's good to keep it the same position for minimum change.

File: packages/daemon/src/app/controller/job.ts [hunk 3]
```diff
@@ -60,19 +61,21 @@ export class JobController {
...
+        let plugins: Partial<Record<PluginTypeI, PluginInfo>>;
+        try {
+          plugins = await this.pipelineService.installPlugins(job, cwd, pyIndex);
+        } catch (err) {
+          job.status = PipelineStatus.FAIL;
+          job.error = err.message;
+          return job.save();
+        }
+        this.pipelineService.startJob(job, cwd, plugins);
```
Comment (yorkie) [inline packages/daemon/src/app/controller/job.ts:77]: Why do we move the `startJob()` outside of the try-clause?

Comment (FeelyChau) [reply packages/daemon/src/app/controller/job.ts:73]: `startJob` do not throw any error.

File: packages/daemon/src/app/controller/job.ts [hunk 4]
```diff
@@ -60,15 +61,17 @@ export class JobController {
         sse.finish();
       }
     } else {
-      try {
-        const plugins = await this.pipelineService.installPlugins(job, cwd, pyIndex);
+      process.nextTick(async () => {
+        let plugins: Partial<Record<PluginTypeI, PluginInfo>>;
```
Comment (yorkie) [inline packages/daemon/src/app/controller/job.ts:65]: Ditto

**Comment by FeelyChau** [comment]
The apis refactored in #380 , so we should close this PR.

**Pull request closed without merging**